content/articles/Integrating-mongodb-with -java/index.md

```diff
@@ -0,0 +1,241 @@
+- [Delete the collection](#Delete-the-collection)
+- [Eclipse versus Intellij Idea in MongoDB integration with Java](#Eclipse-versus-Intellij-Idea-in-MongoDB-integration-with-java)  
+- [Conclusion](#Conclusion)
+ 
+### Prerequisite
+To follow along with this tutorial, you'll need the following: 
+- A basic understanding of the Java language
+- Eclipse IDE installed on your OS- 
+- MongoDB installed on your OS
+- MongoDB Java Connector
+- Java JDK 
+
+### Relational Database Management Systems vs MongoDB
+#### MongoDB is a non-relational database, while Relational Database Management System is a relational database.
+
+Relational databases store data in rows and columns, also known as records. A relational database works by the use of keys to connect information from multiple tables, e.g., foreign keys and primary keys. On the other hand, data get stored in a non-relational database. There are no tables, rows, primary keys, or foreign keys, as there are in a relational database. MongoDB makes use of documents, which get stored in the form of BSON documents.
+MySQL, Oracle, IBM DB2, and Microsoft SQL Server are some of the most popular RDBMS.
+MongoDB, CouchDB, CouchBase, Cassandra, HBase, Redis, Riak, and Neo4J are the popular NoSQL databases.
+
+#### MongoDB uses collections, whereas RDBMS uses tables.
+Collections are how MongoDB organizes documents. In relational databases, collections it makes use of tables.
+[Differences between collections and tables.](https://docs.mongodb.com/manual/core/databases-and-collections/)
+
+
+#### No schema definition is required in MongoDB, but a clear schema is required in RDBMS.
+MongoDB allows unstructured data to be stored that is building your application without first defining the schema. Before adding data to a relational database, you must first define your schema and this affects cchanging the schema structure of a relational database as it'll be expensive and time-consuming. A schema is the structure of the database that is described in a formal language supported by the database management system. Since MongoDB does not require a predefined schema, it is much easier to update as data and requirements are changing.
+
+#### Scaling in MongoDB and Relational DatabasesThe database must be scaled.
+In MongoDB, it's easy to scale, while in RDBMS, the options for scalability are more limited.
+
+### Connecting MongoDB with Java
+
+To connect MongoDB with Java, you'll need to do the following.
+In Eclipse, create a new project. Right-click on the file created, click on properties, and on the Java build path, you'll add the MongoDB Java connector that you downloaded and then click on AddExternalJARS to create a connection between MongoDB and Java. Make sure the MongoDB server is running before using Eclipse or you can follow this link [if you're having any challenges](https://www.mongodb.com/java#:~:text=To%20connect%3A,mongodb%3A%2F%2Flocalhost%3A27017'.)

```

Comment:
`you'll add the MongoDB Java connector that you downloaded`

Where would you find this?